be/src/exec/es/es_predicate.h

```diff
@@ -86,7 +61,25 @@ class VExtLiteral : public ExtLiteral {
         } else if (type == TYPE_DATETIME) {
             vectorized::ColumnViewer<TYPE_DATETIME> viewer(column);
             DCHECK(!viewer.is_null(0));
-            _value = viewer.value(0).to_string();
+            vectorized::TimestampValue datetime_value = viewer.value(0);
+
+            // convert convert default timezone to UTC;
+
+            int year, month, day, hour, minute, second, usec;
+            datetime_value.to_timestamp(&year, &month, &day, &hour, &minute, &second, &usec);
+
+            int64_t timestamp;
+            DateTimeValue ts_value(TIME_DATETIME, year, month, day, hour, minute, second, usec);
+            ts_value.unix_timestamp(&timestamp, TimezoneUtils::default_time_zone);
+
+            DateTimeValue ts_value2;
+            ts_value2.from_unixtime(timestamp, cctz::utc_time_zone());
```

Comment:
this code snippet can be written as an util function dedicated for timezone convertion

Reply:
fixed